src/Generator/ServiceProviderGenerator.php

```diff
@@ -199,6 +200,16 @@ private function generateBodySerializerClosure(Specification $specification): Cl
             );
         }
 
+        if (in_array(VdnApiJsonContentTypeSerializer::MIME_TYPE, $allContentTypes, true)) {
+            $vdnApiJsonContentTypeSerializer = $this->builder->methodCall(
+                $initialStatement,
```

Comment:
then this line should be
```suggestion
                $xmlSerializerInit ?? $formEncodedSerializerInit ?? $jsonSerializerInit ?? $vdnApiJsonContentTypeSerializer ?? $initialStatement,
```